src/Client.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var APIVersion = '2.7'
+var APIVersion = '3'
```

Comment:
@marrony @nathaniel-may With this set to `3`, the new tests I've added will pass, but others will fail. With it set to `2.7`, the only the new tests will fail.

Reply:
For more context: https://github.com/fauna/faunadb-js/pull/295